libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/internal/route/line/MapboxRouteLineUtils.kt

```diff
@@ -388,12 +392,12 @@ object MapboxRouteLineUtils {
                     arrayOfNulls(0)
                 }
 
-                leg.annotation()?.congestion()?.forEachIndexed { index, congestion ->
+                leg.annotation()?.congestionNumeric()?.forEachIndexed { index, congestionNumeric ->
                     val isInAClosure = closureRanges.any { it.contains(index) }
                     val isInRestrictedRange = restrictedRanges.any { it.contains(index) }
                     val congestionValue: String = when {
                         isInAClosure -> RouteConstants.CLOSURE_CONGESTION_VALUE
-                        else -> congestion
+                        else -> CongestionMapper.getCongestionIdentifier(congestionNumeric, congestionRange)
```

Comment:
Why isn't  the`getCongestionIdentifier` function in `MapboxRouteLineUtils` with all the other route line related utility functions?